storage/memory/memory.go

```diff
@@ -57,6 +57,21 @@ func (s *staticIterator) Next() (*openfgapb.Tuple, error) {
 
 func (s *staticIterator) Stop() {}
 
+type StaticObjectIterator struct {
```

Comment:
Let's just move this into the `storage.go` file. It could be useful outside of the scope of the storage/memory package.

Reply:
The static tuple iterator has existed for a while and it's in `memory.go`, too. I don't see why we should move this one if that one hasn't moved